src/views/TradingCompetition/components/PrizesInfo/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Flex } from '@pancakeswap-libs/uikit'
+import PrizesText from './PrizesText'
+import PrizesTable from './PrizesTable'
+
+const Wrapper = styled(Flex)`
+  max-width: 736px;
```

Comment:
You'll see this value in a few places for these central 'info' sections. Was a bit of a yolo value picked by bugs on a whim. I do have the standard 1200px set somewhere else, but I guess he wanted it pulling in more?